docs/contribute/contributing.md

```diff
@@ -59,12 +59,12 @@ To make edits to our documentation via PR, follow these steps:
 
 7. (optional) We recommend running a local test build to ensure the content looks the way you expect.
 
-    In your terminal, run:
+    In your terminal, run the following commands from within your cloned repository:
```

Comment:
Not sure we need to say "from within your cloned repository" (this is a given)

Reply:
Discussed, and agreed to revisit in a future update.